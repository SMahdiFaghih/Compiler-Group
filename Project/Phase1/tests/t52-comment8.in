this one can be tricky//if not careful