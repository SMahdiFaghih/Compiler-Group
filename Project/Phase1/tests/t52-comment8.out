this
T_ID one
T_ID can
T_ID be
T_ID tricky
